index.html

```diff
@@ -7,18 +7,49 @@
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
-      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500;600;700;800&display=swap"
       rel="stylesheet"
     />
     <link rel="stylesheet" href="styles.css" />
+
     <title>portfolio-website</title>
   </head>
 
-  <body>
+  <body id="ani">
+    <section id="mBody">
+      <header id="mHeader">
+        <div id="mHeaderV"></div>
+        <button id="mMenu" title="close" type="button">
+          <img alt="Menu" src="/.png/Union.svg" />
+        </button>
```

Comment:
- [Optional] Good job for doing your menu with DOM based on events fired by the user as your previous code reviewer suggested 👍👍. you missed only your Image path here, correct it please `./.png/Union.svg`👌 and don't use `.` in front of your image directory to make it readable and easily accessible👌